### ISSUE TITLE
Warning: ‘flycheck-disable-checker’ is for interactive use only 

### ISSUE THREAD

**Issue by tarsius** [flycheck/flycheck#1817]
Could you please do something about this warning:

```
lib/flycheck/flycheck.el:2465:43: Warning: ‘flycheck-disable-checker’ is for
    interactive use only; Directly set ‘flycheck-disabled-checkers’ instead
```

Thanks!

**Comment by cpitclaudel** [comment]
Thanks for the report.  Which version of Emacs is this with? My Emacs 27 doesn't show that warning :/ (I think I have a fix, but I want to make sure that it works)

**Comment by tarsius** [comment]
Ah sorry, should have mentioned that. Emacs 28.0.50 aka the master branch.

**Comment by tarsius** [comment]
Friendly ping!

**Comment by tarsius** [comment]
:wave: :grinning:

**Comment by walerian777** [comment]
Hi, the same thing happens for me on Emacs 27.1

**Comment by tarsius** [comment]
Thanks!
